contracts/src/libraries/EigenDABlobVerificationUtils.sol

```diff
@@ -24,6 +25,29 @@ library EigenDABlobVerificationUtils {
         bytes inclusionProof;
         bytes quorumIndices;
     }
+
+    struct SignedCertificate {
+        BlobCertificate blobCertificate;
+        Attestation nonSignerStakesAndSignature;
+    }
+
+    struct BlobCertificate {
+        bytes blobKey;
```

Comment:
We've removed blobKey from cert in the core struct because it can be derived from blob header. Unless we need `blobKey` in the verifier logic, we can probably remove it here too